File: cg_patch_test/FileIO.java
import java.io.File;
import java.io.IOException;

import java.util.Scanner;
import java.io.PrintStream;


public class FileIO{
    public static void main(String[] args){
        try{
            Scanner fin = new Scanner( new File("input.txt") );
            PrintStream fout = new PrintStream( new File("output.txt") );
            String str1 = fin.nextLine();
            fout.println(str1);
        }
        catch(IOException e){
            System.out.println("error");
        }
    }
}

File: cg_patch_test/file_IO.py
#!/usr/bin/python2
fin = open("input.txt", "r")
if not fin:
    print "input.txt not exist"
fout = open("output.txt", "w")
if not fout:
    print "open output.txt error"

s = fin.readline()
fout.write(s)

fin.close()
fout.close()
